### DISCUSSION TITLE
Mumble positional audio support.

### DISCUSSION THREAD

**Discussion by Gliscor** [godotengine/godot-proposals#9683] [Engine Core]
Would be nice to integrate Mumble positional audio support directly into the game engine.

Game engines already supporting this includes Valve's Source Engine, ID's Software ioquake3, open simulator's YEngine.

This would make it easier and faster for developers to enable Mumble positional audio compatibility in their Godot Engine projects.

[Games & Engines that already support Mumble positional audio](https://www.mumble.info/documentation/user/positional-audio/supported-games/)

[Developer documentation to integrate Mumble positional audio](https://www.mumble.info/documentation/developer/positional-audio/link-plugin/)

**Comment by Calinou** [comment]
I feel this really makes more sense as a community add-on, especially since Mumble's popularity has dwindled in the recent years.

Even when Mumble was popular, most people I know disabled positional audio as it was considered distracting.
